grafana-plugin/src/styles/utils.styles.ts

```diff
@@ -26,6 +31,37 @@ export const getUtilStyles = (theme: GrafanaTheme2) => {
     cursorDefault: css`
       cursor: default;
     `,
+
+    overflowChild: css`
+      overflow: hidden;
+      text-overflow: ellipsis;
+      display: -webkit-box;
+      white-space: initial;
+      -webkit-line-clamp: 2;
+      -webkit-box-orient: vertical;
+
+      &--line-1 {
```

Comment:
it looks to much specific, like not a good candidate for reusing from util styles

Reply:
Hmm, what  do you suggest then? This is used for clipping in a few other places, I put it in here so that's being reused

Reply:
I think utils is for very very common classes like `u-position-relative`, `width100` etc, if we start put everything being reused to utils.css then the we come back to the early 2000s when all the styles were placed in the single file `style.css`

Reply:
Can split utils per category then